### ISSUE TITLE
Cant reach report MAP when training from scratch

### ISSUE THREAD

**Issue by davids1974** [pierluigiferrari/ssd_keras#332]
Hello pierluigiferrari,
Thank you for this repository, I've used it for the last couple of months, documentation is also great. 
I also went through all the issues (open and closed).
I have a problem of reaching published MAP value on VOC dataset.
I've made no changes to the training or evaluation code. I train on VOC07 & 12  and test on VOC07. I get MAP of 0.49 after training for 120 epochs with Adam optimizer. Tried SGD with similar results. Strangely my validation loss reaches 4.3 - very similar to the result that you published.
My versions:
ssd_keras - cloned 2d ago
tensorflow-gpu - ver 1.14
keras 2.3.1
keras-applications 1.0.8
keras-preprocessing 1.1.0

![Screenshot from 2020-02-03 15-24-37](https://user-images.githubusercontent.com/55128536/73656684-5a785e80-4699-11ea-933d-5ab7bd6aa9d0.png)

**Comment by aviziskind** [comment]
I encountered a similar problem with low mAP, using a training script based on ssd300_training.ipynb. I found that training with SGD results in a higher mAP than training with the Adam optimizer (ie. the same validation loss results in higher mAP when training with SGD compared to training with Adam, not sure why). Besides that, the only thing I had to change was, for the input arguments to the SSDInputEncoder, I changed the argument `neg_iou_limit` to 0.3 (in the ssd300_training.ipynb, it is 0.5, but the default argument is 0.3, and this default value resulted in faster training for me and higher mAP).
